lib/src/config_util.cc

```diff
@@ -0,0 +1,29 @@
+#include <internal/config_util.hpp>
+#include <cctype>
+
+namespace hocon {
+
+    bool is_whitespace(char c) {
+        switch (c) {
+            case ' ':
+            case '\n':
+            // TODO: these complain about char overflow, what do.
```

Comment:
Perhaps use `u8_to_u32_iterator` (http://www.boost.org/doc/libs/1_58_0/libs/regex/doc/html/boost_regex/ref/internal_details/uni_iter.html) to enumerate as u32 characters?